.github/workflows/ci.yaml

```diff
@@ -40,6 +40,8 @@ jobs:
         run: make install
       - name: Run tests
         run: bin/test.sh
+      - name: Run retry
+        run: DL_QUEUE_CONSTANT_RETRY=false poetry run python -m jbi.retry.py
```

Comment:
Don't know how much this matters in the grand scheme of things, but would it be better to explicitly use the Python in our .venv, like we do in the docker container? Don't know if `poetry run` does any side effect magic 🤔.